spring-boot-project/spring-boot-actuator-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -162,7 +171,10 @@
         "OUT_OF_SERVICE",
         "UP",
         "UNKNOWN"
-      ]
+      ],
```

Comment:
This shouldn't be required as the annotation processor is supposed to pick things up. I've opened #18184

Reply:
The issue is fixed so I'll rebase and polish this part.